commands/acl-setuser.md

```diff
@@ -57,12 +73,19 @@ This is a list of all the supported Redis ACL rules:
 * `-<command>`. Like `+<command>` but removes the command instead of adding it.
 * `-@<category>`: Like `+@<category>` but removes all the commands in the category instead of adding them.
 * `nocommands`: alias for `-@all`. Removes all the commands, the user will no longer be able to execute anything.
+
+### User management rules
+
+* `on`: set the user as active, it will be possible to authenticate as this user using `AUTH <username> <password>`.
+* `off`: set user as not active, it will be impossible to log as this user. Please note that if a user gets disabled (set to off) after there are connections already authenticated with such a user, the connections will continue to work as expected. To also kill the old connections you can use `CLIENT KILL` with the user option. An alternative is to delete the user with `ACL DELUSER`, that will result in all the connections authenticated as the deleted user to be disconnected.
 * `nopass`: the user is set as a "no password" user. It means that it will be possible to authenticate as such user with any password. By default, the `default` special user is set as "nopass". The `nopass` rule will also reset all the configured passwords for the user.
 * `>password`: Add the specified clear text password as an hashed password in the list of the users passwords. Every user can have many active passwords, so that password rotation will be simpler. The specified password is not stored as clear text inside the server. Example: `>mypassword`.
 * `#<hashedpassword>`: Add the specified hashed password to the list of user passwords. A Redis hashed password is hashed with SHA256 and translated into a hexadecimal string. Example: `#c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2`.
 * `<password`: Like `>password` but removes the password instead of adding it.
 * `!<hashedpassword>`: Like `#<hashedpassword>` but removes the password instead of adding it.
-* reset: Remove any capability from the user. It is set to off, without passwords, unable to execute any command, unable to access any key.
+* `(<rule list>)`: (Available in Redis 7.0 and later) Create a new selector to match rules against. Selectors are evaluated after the user permissions, and are evaluated according to the order they are defined. If a command matches either the user permissions or any selector, it is allowed. See [selectors](/topics/acl#selectors) for more information.
+* `clearselectors`: (Available in Redis 7.0 and later) Delete all of the selectors attached to the user.
+* `reset`: Remove any capability from the user. It is set to off, without passwords, unable to execute any command, unable to access any key.
```

Comment:
How about uppercase for the pure tokens `CLEARSELECTORS`, `RESET` here as well as and `ON`, `OFF` and `NOPASS` above?

Reply:
Not sure these are pure tokens, since they aren't part of the command syntax. We would probably want all keywords to uppercase then throughout the doc, which I don't really want to mess with git history in this PR.

Reply:
OK. Leave them lowercase.